commcare_connect/opportunity/visit_import.py

```diff
@@ -256,7 +260,16 @@ def _bulk_update_payments(opportunity: Opportunity, imported_data: Dataset) -> P
             except ValueError:
                 invalid_rows.append((row, "amount must be an integer"))
             else:
-                payments[username] = amount
+                payments[username] = {"amount": amount}
+                try:
+                    if date_paid_raw:
+                        date_paid = datetime.datetime.strptime(date_paid_raw, "%Y-%m-%d").date()
```

Comment:
is there a reason to use this instead of one of the more flexible date parsers?

Reply:
What other flexible date parsers are you referring to?

Reply:
I was thinking [this django util](https://docs.djangoproject.com/en/5.0/ref/utils/#module-django.utils.dateparse) which is mostly the same but slightly nicer to use, and a bit more flexible (mostly around digits for months and days)